make_plant_parts_eia.py

```diff
@@ -1512,16 +1517,11 @@ def generate_master_unit_list(self, qual_records=True, clobber=False):
                         part_df, part_name, qual_record
                     )
             plant_parts_df = plant_parts_df.append(part_df, sort=True)
-        # clean up, add additional columns, and drop duplicates
+        # clean up, add additional columns
         self.plant_parts_df = (
             self.add_additonal_cols(plant_parts_df)
             .pipe(pudl.helpers.organize_cols, FIRST_COLS)
             .pipe(self._clean_plant_parts)
-            .pipe(self.dedup_on_category,
-                  category_name='ownership',
-                  base_cols=[x for x in plant_parts_df.columns if x not in [
-                      'record_id_eia', 'ownership', 'appro_record_id_eia']],
-                  sorter=['owned', 'total'])
```

Comment:
I got rid of this because this was the previous way that I had thought I was dealing with this problem. It was removing a portion of the records which I am now labeling as duplicates, but not all of them for some reason. I like the label it and deal with it later solution better so I removed this.